build.sbt

```diff
@@ -0,0 +1,19 @@
+scalaVersion := "2.12.4"
+
+lazy val root = project
+  .in(file("."))
+  .aggregate(unidocPlugin)
```

Comment:
I'm not sure we need a 'root' and a 'plugin', I think in this case the root project can be the plugin directly.

(in case of akka-paradox we're publishing the plugin and the theme separately, for unidoc support I don't think we need a theme)